scripts/stacked_barplot_fishers_test.R

```diff
@@ -0,0 +1,19 @@
+---
+title: "stacked_barplot_fishers_test"
+author: "Selam Mehreteab(smehrete@ucsc.edu)"
+date: "`r format(Sys.time(), '%B %d, %Y')`"
+purpose: "Run fishers test on stacked barplot"
+---
+
+library(rstatix)
+
+xtab <- as.table(rbind( c(209,51), c(104,12), c(180,79), c(212,92), c(636,298), c(18,1), c(7,3)))
```

Comment:
```suggestion
xtab <- as.table(rbind( c(209,51), c(104,12), c(180,79), c(212,92), c(636,298), c(18,1), c(7,3), c(44,0)))
```
It appears that the intron retention numbers are missing?